### ISSUE TITLE
Staff issues

### ISSUE THREAD

**Issue by cdf75ubc** [polyrhythm-project/polyrhythm-website#29]
ex. T327, Bartok, on VHV staffs have different spacing for the strings from that of SIB .pdf.

**Comment by cdf75ubc** [comment]
ex. T328, Bartok, on VHV staffs have different spacing, but OK.

**Comment by craigsapp** [comment]
Is this what you see for T327?:
https://polyrhythm.humdrum.org/example/?id=327

<img width="648" alt="Screen Shot 2020-01-18 at 17 03 23" src="https://user-images.githubusercontent.com/3487289/72672774-7ed70880-3a14-11ea-825b-f396f22bac92.png">

No spacings of the staves are used in the Verovio (VHV) rendering of the notation.  The MusicXML file has some basic spacing but not complete spacing information for the general case (it gives numbers which basically function as minimum spacing, but there will be allowance for expanding spacing as needed).

There are ways of fixing, but not of great importance.  There is an option in verovio to set a minimum distance for staff and system separation.  These parameters could be applied to all examples to avoid cases such as the tight staff distance between the cellos and bases.  

The distance between the violin I and violin II is related to the dynamic marking.  Without looking at the Sibelius PDF or the original notation, I am sure that the `pp` dynamic is not placed directly below the notes, but rather to the left of the stem to minimize the distance between the two staves.   For fixing this there are two problems: (1) verovio (the notation renderer) does not allow for such placement yet, and (2) there is a hack that can be done (but this will be a manual edit to the Humdrum data which we can wait on doing until a more refined proofreading is needed).

**Comment by cdf75ubc** [comment]
ex. T326, Bartok, viola, bar 1: (just as a follow up example)
<img width="266" alt="Screen Shot 2020-01-20 at 11 01 19 AM" src="https://user-images.githubusercontent.com/59900770/72751702-2a976a00-3b75-11ea-8c08-06a0e20befa9.png">

desired:

<img width="291" alt="Screen Shot 2020-01-20 at 11 18 05 AM" src="https://user-images.githubusercontent.com/59900770/72752248-8d3d3580-3b76-11ea-8acc-f8e7d705d119.png">

**Comment by cdf75ubc** [comment]
<img width="278" alt="Screen Shot 2020-01-20 at 11 19 16 AM" src="https://user-images.githubusercontent.com/59900770/72752312-bbbb1080-3b76-11ea-9365-f5b49b3c9da7.png">

**Comment by cdf75ubc** [comment]
@craigsapp ex. T344, Bartok, in a case like this, it is not clear to which instrument the articulation is applied. Question: should I continue reporting this issue? Thanks!

PDF
<img width="597" alt="Screen Shot 2020-01-29 at 6 18 40 PM" src="https://user-images.githubusercontent.com/59900770/73414601-292c1700-42c4-11ea-9bf1-f8de696cff62.png">

VHV
<img width="549" alt="Screen Shot 2020-01-29 at 6 19 45 PM" src="https://user-images.githubusercontent.com/59900770/73414609-2f21f800-42c4-11ea-85ce-1ff6791df81c.png">

**Comment by cdf75ubc** [comment]
on the other hand, sometimes VHV fixes SIB .pdf poor staves spacing:

PDF

<img width="193" alt="Screen Shot 2020-01-29 at 6 22 46 PM" src="https://user-images.githubusercontent.com/59900770/73414699-685a6800-42c4-11ea-95a5-5c572cb4c276.png">

VHV

<img width="169" alt="Screen Shot 2020-01-29 at 6 22 57 PM" src="https://user-images.githubusercontent.com/59900770/73414704-6e504900-42c4-11ea-9e63-094027c62919.png">

**Comment by craigsapp** [comment]
For T344, the note stems getting too close to each other on adjacent staves is a known problem in verovio, with an issue discussing a similar problem:  https://github.com/rism-ch/verovio/issues/977

For the second problem where verovio is better: verovio automatically adjusts the staves to force a minimum distance between, so it will automatically separate staves if their contents collides.  It is not always perfect such as in the stems interlocking, which is technically correct, but not musically correct.

**Comment by cdf75ubc** [comment]
@craigsapp , ex. T358 looks like a staff "horizontal" spacing problem, I am listing it under this issue, please check, it is difficult to proofread it. VHV below first and PDF on the bottom. 

<img width="947" alt="Screen Shot 2020-01-31 at 5 09 03 PM" src="https://user-images.githubusercontent.com/59900770/73584339-7804b880-444c-11ea-81ae-d8caad6c701f.png">

<img width="1393" alt="Screen Shot 2020-01-31 at 5 08 45 PM" src="https://user-images.githubusercontent.com/59900770/73584340-7804b880-444c-11ea-8165-38ef7addf1af.png">

**Comment by craigsapp** [comment]
The problem is most likely related to a rhythmic representation problem (which could possibly come from different sources so I will have to study the problem further).

**Comment by cdf75ubc** [comment]
@epoudrier : same for the example above, I have listed it as "ongoing", besides SIB .pdf has a few problems regarding 2-voice writing, mainly ties for the lower voice of bass clef that will need to be proofread later.

**Comment by cdf75ubc** [comment]
ex. T596, Honneger, staff vertical spacing causes accents to collide with tones.

PDF

<img width="309" alt="Screen Shot 2020-02-05 at 11 05 14 AM" src="https://user-images.githubusercontent.com/59900770/73873938-71d55a00-4807-11ea-8c4e-2bdbe620351d.png">


VHV


<img width="472" alt="Screen Shot 2020-02-05 at 11 05 27 AM" src="https://user-images.githubusercontent.com/59900770/73873950-77cb3b00-4807-11ea-84a5-0b25c1c11cb1.png">

**Comment by cdf75ubc** [comment]
ex. T597, please notice staff vertical spancing for piano:

PDF

<img width="900" alt="Screen Shot 2020-02-05 at 11 54 36 AM" src="https://user-images.githubusercontent.com/59900770/73877895-5b7ecc80-480e-11ea-9f51-7dd3757688ba.png">


VHV


<img width="1261" alt="Screen Shot 2020-02-05 at 11 54 48 AM" src="https://user-images.githubusercontent.com/59900770/73877907-620d4400-480e-11ea-9823-eea76d7abfec.png">

**Comment by epoudrier** [comment]
@cdf75ubc make sure that the description clearly identifies the problem. E.g., for T597 above, I cannot tell what the problem is from the description.

**Comment by cdf75ubc** [comment]
@epoudrier : Right! I apologize. @craigsapp told me about issue #29 that : "verovio automatically adjusts the staves to force a minimum distance between" , my point was that, in this case, ex. T597, for the piano grand staff, staves were too far apart without any reason, and that the PDF looked good. Thanks for pointing it out!

**Comment by cdf75ubc** [comment]
ex. T672, Hindemith, as in T358, the same problem happens here: I am calling it a "Staff Horizontal Spacing" issue, and listing it under issue #29. Please see screenshots below.

PDF

<img width="945" alt="Screen Shot 2020-02-20 at 3 31 28 PM" src="https://user-images.githubusercontent.com/59900770/74989867-7f3d3780-53f6-11ea-8e1c-1de0856773ed.png">




VHV

<img width="1254" alt="Screen Shot 2020-02-20 at 3 33 53 PM" src="https://user-images.githubusercontent.com/59900770/74989885-8a906300-53f6-11ea-960e-e543f023e476.png">
